1.0/docs/start/psk/creating-a-page.md

```diff
@@ -0,0 +1,211 @@
+   snippet below.
+
+{% highlight html %}
+...
+<!-- Drawer Content -->
+<paper-menu class="list" attr-for-selected="data-route" selected="{{route}}" on-iron-select="onMenuSelect">
+  <a data-route="home" href="/">
+    <iron-icon icon="home"></iron-icon>
+    <span>Home</span>
+  </a>
+...
+{% endhighlight %}
+
+Each navigation menu item consists of an `a` element with two children: 
+`iron-icon` and `span`.
+
+* `iron-icon` displays a single icon. 
+* `span` displays the text next to the icon.
+   
+1. Add the following new navigation item to the bottom of the menu.
+
+{% highlight html %}
+<a data-route="books" href="/books">
+  <iron-icon icon="book"></iron-icon>
+  <span>Books</span>
+</a>
+{% endhighlight %}
+
+Your menu should now look like the code below (some code has been omitted).
+
+{% highlight html %}
+...
+<!-- Drawer Content -->
+<paper-menu class="list" attr-for-selected="data-route" selected="{{route}}" on-iron-select="onMenuSelect">
+  <a data-route="home" href="/">
+    <iron-icon icon="home"></iron-icon>
+    <span>Home</span>
+  </a>
+  ...
+  <a data-route="contact" href="/contact">
+    <iron-icon icon="mail"></iron-icon>
+    <span>Contact</span>
+  </a>
+  <a data-route="books" href="/books">
+    <iron-icon icon="book"></iron-icon>
+    <span>Books</span>
+  </a>
+</paper-menu>
+...
+{% endhighlight %}
+
+If you view the app now, you should see your new item in the navigation
+menu, but the link does not point to a valid page yet. Proceed to the 
+next section to do that now.
+
+<!-- image with new nav item -->
+
+## Adding content
+
+In the previous section you added a navigation menu item to enable the 
+user to navigate to a new page. Now, you add the content for that new page.
+
+1. Open `app/index.html` and find the main content. The beginning of the main
+   content section is provided in the code snippet below (some code omitted for
+   brevity).
+
+{% highlight html %}
+<div class="content">
+  <iron-pages attr-for-selected="data-route" selected="{{route}}">
+
+    <section data-route="home">
+      <paper-material elevation="1">
+        <my-greeting></my-greeting>
+
+        <p class="paper-font-subhead">You now have:</p>
+        <my-list></my-list>
+        ...
+      </paper-material>
+    </section>
+
+    <section data-route="users">
+      <paper-material elevation="1">
+        <h2 class="paper-font-display2">Users</h2>
+        <p>This is the users section</p>
+        <a href="/users/Rob">Rob</a>
+      </paper-material>
+    </section>
+    ...
+{% endhighlight %}
+
+* Each page of the application is a `section` element. 
+  * The `data-route` attribute is an identifier for the routing system. 
+  You'll set that up for your new page in the next section.
+* The `paper-material` element creates a card which floats on top of the
+  main content area. All text, images, and other content should always
+  be contained within one of these cards.
```

Comment:
:heavy_check_mark: clarified that structuring each page as a `<section>` is a PSK design pattern and that the `<paper-material>` stuff is MD